src/sentry/static/sentry/app/components/sidebar/index.tsx

```diff
@@ -164,15 +176,6 @@ class Sidebar extends React.Component<Props, State> {
     });
   };
 
-  // Hide slideout panel
-  hidePanel = () => {
-    if (this.state.currentPanel === '') {
-      return;
-    }
-
-    this.setState({currentPanel: ''});
-  };
-
```

Comment:
imo it'd be cleaner to keep this method and have this method call the store?

Reply:
yeah I think you're right